index.js

```diff
@@ -159,14 +187,20 @@ module.exports = {
         if (!addMemberSpec(content))
           return cb(new Error(addMemberSpec.errorsString))
 
-        publish(content, (err, msg) => {
+        const invitations = {
+          purpose: 'invitations',
+          feedFormat: 'classic',
```

Comment:
```suggestion
          feedFormat: 'classic',
          metadata: {
            encryption: { key: <base64>, format: 'curve25519' }
          }
```

Reply:
We need a spec for this. Or, we need this put in some existing spec.

Reply:
Also, how do I convert from this to a `DHKeys`? It seems like DHKeys needs `{ public, secret }`

Reply:
We are not converting, these will be raw DH keys. And we are only publishing the public part!!!  the private part must be persisted by you

Reply:
https://github.com/ssbc/ssb-meta-dm-spec/issues/1